src/ensembl/src/content/app/browser/browser-image/BrowserImage.tsx

```diff
@@ -183,7 +186,10 @@ export const BrowserImage: FunctionComponent<BrowserImageProps> = (
           ref={props.browserRef}
         />
         <BrowserCogList />
-        <ZmenuController browserRef={props.browserRef} />
+        <ZmenuController
+          browserRef={props.browserRef}
+          changeHighlightedTrack={props.changeHighlightedTrack}
```

Comment:
Just a thought: what if we connected ZmenuController to redux and let it itself get the `changeHighlightedTrack` function?

I am not sure whether there are any guidelines regarding when to connect to redux directly and when to receive the prop from the parent. It just feels that changing tracks is not `BrowserImage`'s responsibility.

Reply:
That's a fair point and I think it's better to connect ZmenuController itself to redux.